.github/workflows/test-wheel-linux.yml

```diff
@@ -180,6 +180,23 @@ jobs:
           host-platform: ${{ inputs.host-platform }}
           cuda-version: ${{ inputs.cuda-version }}
 
+      - name: Set up compute-sanitizer
+        run: |
+          if [[ "${{ inputs.python-version }}" == "3.12" && "${{ inputs.local-ctk }}" == 1 ]]; then
+            COMPUTE_SANITIZER="${CUDA_HOME}/bin/compute-sanitizer"
+            COMPUTE_SANITIZER_VERSION=$(${COMPUTE_SANITIZER} --version | grep -Eo "[0-9]{4}\.[0-9]\.[0-9]" | sed -e 's/\.//g')
+            SANITIZER_CMD="${COMPUTE_SANITIZER} --target-processes=all --launch-timeout=0 --tool=memcheck --error-exitcode=1"
+            if [[ "$COMPUTE_SANITIZER_VERSION" -ge 202111 ]]; then
+                SANITIZER_CMD="${SANITIZER_CMD} --padding=32"
+            fi
+            echo "CUDA_PYTHON_SANITIZER_RUNNING=1" >> $GITHUB_ENV
```

Comment:
Could you document this test-time env in https://github.com/NVIDIA/cuda-python/blob/main/cuda_bindings/docs/source/environment_variables.md?

Reply:
bdfb547c6dc57c0bfdeeb4fa36859a28a00dc624